### ISSUE TITLE
Storyline popping up while trying to enter Torghast

### ISSUE THREAD

**Issue by Flanell88** [Total-RP/Storyline#79]
Same issue as 2020 is up again: storyline window is popping up while trying to enter Torghast overlapping the default Blizzard UI to pick the layer.

**Comment by Solanya** [comment]
Indeed, my mistake, broke it while experimenting with some changes from 10.0. 😅 
Will be fixed again for 10.0.2
